hack/patch_node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
```

Comment:
Can you explain why this `echo` is needed?

Reply:
just to split between output of the previous stdout and this one, so the output will be nicer.